src/popup/pages/InviteClaim.vue

```diff
@@ -6,26 +6,30 @@
 import { defineComponent, onMounted } from 'vue';
 import { decode } from '@aeternity/aepp-sdk';
 import { useStore } from 'vuex';
-import { useRouter } from 'vue-router';
+import { useRouter, useRoute } from 'vue-router';
 import { useModals, useAeSdk } from '../../composables';
 import { ROUTE_ACCOUNT } from '../router/routeNames';
 
 export default defineComponent({
-  props: {
-    secretKey: { type: String, required: true },
-  },
-  setup(props) {
+  setup() {
     const store = useStore();
     const router = useRouter();
+    const route = useRoute();
     const { getAeSdk } = useAeSdk({ store });
     const { openDefaultModal } = useModals();
 
     onMounted(async () => {
       await getAeSdk();
 
       try {
-        // sg_ prefix was chosen as a dummy to decode from base58Check
-        await store.dispatch('invites/claim', decode(`sg_${props.secretKey}`));
+        // nm_ prefix was chosen as a dummy to decode from base58Check
+        // The secretKey can be retrieved from the URL in two different ways:
+        // current: /invite#${secretKey}
+        // legacy: /invite/${secretKey}
```

Comment:
Would be good to have it covered with tests

Reply:
We will do it in the future.